kfdef/kfctl_aws.yaml

```diff
@@ -130,6 +130,7 @@ spec:
       overlays:
       - istio
       - application
+      - aws
```

Comment:
https://github.com/kubeflow/manifests/blob/dd3b85ed462ec3241150730b8b0d0df2e0bb5cee/kfdef/source/master/kfctl_aws.yaml

Can you update this file. See how does it work. https://github.com/kubeflow/manifests/tree/dd3b85ed462ec3241150730b8b0d0df2e0bb5cee/kfdef/source

Reply:
Got it, I addressed them in the latest revision.

Reply:
I also updated cognito yaml file, let me know if that's unnecessary.